src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final double arrayVolumeAdd = 1.5;
```

Comment:
1. Constant naming should be PROPERLY_CASED
2. Name of constant/variable is usually a noun, verb is suitable for methods
```suggestion
    private static final double GROW_MULTIPLIER = 1.5;
```